### DISCUSSION TITLE
LEDVANCE Produkte einbinden

### DISCUSSION THREAD

**Discussion by hubertmeier** [OpenCCU/OpenCCU#1872] [🇩🇪 Fragen&Antworten] [answered]
Hallo,

gibt es eine Möglichkeit, die sehr preisgünstigen Ledvance Bluetooth und WLAN Produkte in die Raspberrymatic einzubinden? Der Raspberry könnte ja per Bluetooth direkt mit den Lampen gepaart werden, die Reichweite ist natürlich bescheiden. Die WLAN-Produkte würden ja überall im Netz funktionieren. Ich meine ein direktes Ansprechen der Geräte über die Homematic oder über ein Addon ohne Cloudanbindung beim Hersteller.

Ansonsten, tolle Arbeit. vielen Dank.

**Comment by jens-maus** [comment, marked as answer]
> gibt es eine Möglichkeit, die sehr preisgünstigen Ledvance Bluetooth und WLAN Produkte in die Raspberrymatic einzubinden?

Aus dem Stehgreif wüsste ich jetzt nicht wie genau. Das müsste sicherlich von Einzelanwendung zu Einzelanwendung man sich einfach mal genauer anschauen. Aber direkt in die WebUI werden sich diese Third-Party Produkte sicherlich nicht einfach so integrieren lassen und wenn dann ggf. nur über das CUxD Addon das es für eine CCU/RaspberryMatic ja gibt.
